app/assets/javascripts/analytics_modules/ga4-form-setup.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analyticsModules || {}
```

Comment:
I think these lines could be slightly simplified by using the `||=` operator:
```suggestion
window.GOVUK ||= {}
window.GOVUK.analyticsGa4 ||= {}
window.GOVUK.analyticsGa4.analyticsModules ||= {}
```

Reply:
> I think these lines could be slightly simplified by using the `||=` operator:

You are right about that. However (for whatever reason) these are written like this on all our JS modules across GOVUK so I'd be inclined to leave it like this. I have no idea if it is just convention or if something disastrous would happen.